ocp_install_env.sh

```diff
@@ -15,6 +16,41 @@ export EXTERNAL_SUBNET="192.168.111.0/24"
 #
 export OPENSHIFT_RELEASE_IMAGE="registry.svc.ci.openshift.org/kni/release:4.1.0-rc.5-kni.1"
 
+function extract_installer() {
+    local release_image
+    local outdir
+
+    release_image="$1"
+    outdir="$2"
+
+    extract_dir=$(mktemp -d "installer--XXXXXXXXXX")
+
+
+    echo "${PULL_SECRET}" > "${extract_dir}/pullsecret"
+    oc adm release extract --registry-config "${extract_dir}/pullsecret" --command=openshift-install --to "${extract_dir}" "${release_image}"
+    mv "${extract_dir}/openshift-install" "${outdir}"
+    export OPENSHIFT_INSTALLER="${outdir}/openshift-install"
+
+    rm -rf "${extract_dir}"
+}
+
+function clone_installer() {
+  # Clone repo, if not already present
+  if [[ ! -d $OPENSHIFT_INSTALL_PATH ]]; then
+    sync_repo_and_patch go/src/github.com/openshift-metalkube/kni-installer https://github.com/openshift-metalkube/kni-installer.git
+  fi
+}
+
+function build_installer() {
+  # Build installer
+  pushd .
+  cd $OPENSHIFT_INSTALL_PATH
+  RELEASE_IMAGE="$OPENSHIFT_RELEASE_IMAGE" TAGS="libvirt ironic" hack/build.sh
```

Comment:
I don't remember setting RELEASE_IMAGE before.  What does this do?  What would happen if it wasn't set?

Reply:
See https://github.com/openshift-metal3/dev-scripts/pull/558/files/088d450a2e33cf79d6694c3fa1e0debac1de91ed#diff-2407feeeedf4de6f5b077ee1746f344b

Reply:
See https://github.com/openshift-metal3/dev-scripts/pull/558#discussion_r286366618

Reply:
thanks!